src/transformers/file_utils.py

```diff
@@ -1115,9 +1115,10 @@ def _prepare_output_docstrings(output_type, config_class):
         >>> model = {model_class}.from_pretrained('{checkpoint}')
 
         >>> inputs = tokenizer("Hello, my dog is cute", return_tensors="jax")
```

Comment:
(nit)
Think we should return `np` tensors instead of `jax`, as we have seen during the sprint that if pre-processing functions which use JAX are called in multi-processing code such as our `dataset.map` it creates issues. It's not an issue here but would be nice to if always encourage to use `numpy` when pre-processing.

Reply:
Agree! Think this should be a follow-up PR to correct this everywhere (examples, notebooks, blog posts, ...)